brands/shop/tyres.json

```diff
@@ -66,6 +66,17 @@
       "shop": "tyres"
     }
   },
+  "shop/tyres|Eurotyre~(Portugal)": {
+    "countryCodes": ["pt"],
+    "matchTags": ["shop/car_repair"],
+    "tags": {
+      "brand": "Eurotyre",
+      "brand:wikidata": "Q3060871",
+      "brand:wikipedia": "pt:Eurotyre",
```

Comment:
Eurotyre don't exist (yet) in[ wikipedia:pt](https://pt.wikipedia.org/wiki/Special:Search?search=Eurotyre&go=Go&ns0=1)